i18n/locales/fr.json

```diff
@@ -24,7 +24,8 @@
       "easyDescription": "L'interface est simple, ne nécessite aucun compte et détecte automatiquement les autres appareils."
     },
     "mentioned": "Le monde entier en parle",
-    "get": "Obtenez LocalSend"
+    "get": "Obtenez LocalSend",
+    "video": "Regarder une présentation vidéo de LocalSend sur YouTube"
```

Comment:
Happy to generate translations for other languages with Google Translate or something, if that sounds like an acceptable default?

Reply:
I had some translators not happy about auto generation, so let's keep them in English :)